src/screens/OrganizationTags/OrganizationTags.tsx

```diff
@@ -0,0 +1,511 @@
+  const { t } = useTranslation('translation', {
+    keyPrefix: 'organizationTags',
+  });
+  const { t: tCommon } = useTranslation('common');
+
+  const [createTagModalIsOpen, setCreateTagModalIsOpen] = useState(false);
+
+  const { orgId: currentUrl } = useParams();
+  const navigate = useNavigate();
+  const [after, setAfter] = useState<string | null | undefined>(null);
+  const [before, setBefore] = useState<string | null | undefined>(null);
+  const [first, setFirst] = useState<number | null>(5);
+  const [last, setLast] = useState<number | null>(null);
+  const [tagSerialNumber, setTagSerialNumber] = useState(0);
+
+  const [tagName, setTagName] = useState<string>('');
+
+  const [removeUserTagId, setRemoveUserTagId] = useState(null);
+  const [removeTagModalIsOpen, setRemoveTagModalIsOpen] = useState(false);
+
+  const showCreateTagModal = (): void => {
+    setTagName('');
+    setCreateTagModalIsOpen(true);
+  };
+
+  const hideCreateTagModal = (): void => {
+    setCreateTagModalIsOpen(false);
+  };
+
+  const {
+    data: orgUserTagsData,
+    loading: orgUserTagsLoading,
+    error: orgUserTagsError,
+    refetch: orgUserTagsRefetch,
+  }: {
+    data?: {
+      organizations: InterfaceQueryOrganizationUserTags[];
+    };
+    loading: boolean;
+    error?: ApolloError;
+    refetch: () => void;
+  } = useQuery(ORGANIZATION_USER_TAGS_LIST, {
+    variables: {
+      id: currentUrl,
+      after: after,
+      before: before,
+      first: first,
+      last: last,
+    },
+  });
+
+  const [create, { loading: createUserTagLoading }] =
+    useMutation(CREATE_USER_TAG);
+
+  const createTag = async (e: ChangeEvent<HTMLFormElement>): Promise<void> => {
+    e.preventDefault();
+
+    try {
+      const { data } = await create({
+        variables: {
+          name: tagName,
+          organizationId: currentUrl,
+        },
+      });
+
+      if (data) {
+        toast.success(t('tagCreationSuccess'));
+        orgUserTagsRefetch();
+        setTagName('');
+        setCreateTagModalIsOpen(false);
+      }
+    } catch (error: unknown) {
+      /* istanbul ignore next */
+      if (error instanceof Error) {
+        toast.error(error.message);
+        console.log(error.message);
+      }
+    }
+  };
+
+  const [removeUserTag] = useMutation(REMOVE_USER_TAG);
+  const handleRemoveUserTag = async (): Promise<void> => {
+    try {
+      await removeUserTag({
+        variables: {
+          id: removeUserTagId,
+        },
+      });
+
+      orgUserTagsRefetch();
+      toggleRemoveUserTagModal();
+      toast.success(t('tagRemovalSuccess'));
+    } catch (error: unknown) {
+      /* istanbul ignore next */
+      if (error instanceof Error) {
+        toast.error(error.message);
+        console.log(error.message);
+      }
+    }
+  };
+
+  if (createUserTagLoading || orgUserTagsLoading) {
+    return <Loader />;
+  }
+
+  if (orgUserTagsError) {
+    console.log(orgUserTagsError.message);
```

Comment:
i think this `console.log` can be removed?

Reply:
yes, thank you